src/plat/pc99/linker.lds

```diff
@@ -27,10 +27,17 @@ nodeSkimScratchOffset = nodeSkimScratch - node_info;
 
 KERNEL_OFFSET = KLOAD_VADDR - KLOAD_PADDR;
 
+PHDRS {
```

Comment:
I needed to explicitly define the segments here because the default segments being created by the linker had the first PT_LOAD segment not starting from 0 ELF file offset which would cause a page worth of 0 padding right after the segment headers.